website/files/utils.py

```diff
@@ -16,12 +18,19 @@ def copy_files(src, target_node, parent=None, name=None):
     cloned.save()
 
     if src.is_file and src.versions.exists():
-        cloned.versions.add(*src.versions.all())
-        most_recent_fileversion = cloned.versions.select_related('region').order_by('-created').first()
+        fileversions = src.versions.select_related('region').order_by('-created')
+        most_recent_fileversion = fileversions.first()
         if most_recent_fileversion.region != target_node.osfstorage_region:
+            # add all original version except the most recent
+            cloned.versions.add(*fileversions[1:])
+            # setting the id to None and calling save generates a new object
```

Comment:
The way this copies the old fileversion is a little odd - got it from [the django docs](https://docs.djangoproject.com/en/dev/ref/models/instances/#how-django-knows-to-update-vs-insert) referenced from [this stackoverflow answer](https://stackoverflow.com/questions/4733609/how-do-i-clone-a-django-model-instance-object-and-save-it-to-the-database) - basically by setting the id to `None` it creates a new object instead of updating the old one.

I'm not totally set on this method over just doing `FileVersion.objects.create()` with all the other fileversion's attributes, as that maybe seems clearer?

Reply:
I'm not familiar w/ Django models, but how is the `src.clone()` method on line 12 implemented?  Is that a DRM-provided method, or is that something we added?  If the latter, what approach did we use?

Reply:
AHHHHH I just assumed that was `Node` specific, but it's something we have on `BaseModel`! Woo! Thatttt does pretty much this but better. Thanks @felliott!!

PS: also fixing the failing test